app/models/cart.rb

```diff
@@ -0,0 +1,6 @@
+class Cart < ApplicationRecord
+  belongs_to :user
+  has_many :cart_items
```

Comment:
don't we need to destroy all the `cart_items` after destroying the parent `cart`? 
```suggestion
  has_many :cart_items, dependent: :destroy
```

Reply:
Fixed it bhai. Thank you.